### DISCUSSION TITLE
Notes Template Update

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/VC-Community-Docs#133] [Coffees]
Hey everyone! I'm following up on a topic discussed in the meeting this afternoon re: updating the notes template.

Please see PR #132 for what I started. I took a crack at adding in directions for a Google Docs workflow and sharing the notes with the Room Leaders. This is **all** very much open for suggestions so either comment here or in the PR!

I had no idea what to put down for the message at the end to encourage people to engage in Slack. HALP!

---

Below is a copy of the Issue #131 opened for this.

# Description

Rework template to streamline notetaker workflow. We can achieve this with slight changes to the content and then creating a Google Doc which can be copied for each new breakout room session.

## Content

- [ ] Add line to end of document encouraging members to continue the conversation in Slack
- [x] Change `MAIN CONVERSATION` to `TOPIC #1` and add a few more topics
- [x] Fix grammar/typos/spelling if necessary
- [ ] Create Google Doc template

## Format

Copy content of the template to a Google Doc. Allow access for anyone with the link and paste the link back into the `Notetakers/README.md` file.

**Comment by BekahHW** [comment]
I would love to get more feedback here bc I heard two things in the meeting today:

1. We need less structure;
2. We need more structure.

Maybe there was an overwhelming response and I didn't quite get it, but I want to make sure this is right for most of our group--maybe even an anonymous poll in slack could make this clearer. But also, I'm not sure this needs to be an either-or thing. I think there are other ways to supplement notetaking or support it. And I'm very much in favor of changing the process once after a lot of discussion.

## Questions
1. Do notetakers mostly want a google doc--which adds one more place to keep track of?
2. Do we want to record all the topics and detailed notes or stick to intros, summaries, and links?
3. Do we want to let the notetakers do what they feel comfortable with, whether that's using a shared template or YOLOing notes?

**Reply by meg-gutshall** [reply]
I think you're right in that there were people of both camps in yesterday's discussion. My thinking to address these dichotomous needs is to create more structure so it's available for people who want it while also making clear that using the pre-made template/workflow isn't required if you already have your own workflow set up, for instance, like @clandau.

## Questions

1. I don't understand what you mean by "which adds one more place to keep track of." Are you referring to using Google Drive as a whole or another copy of the notes? Maybe we should have asked yesterday what people are currently using to take notes. Do you think we should create a Polly in #vc-room-leaders-notetakers to find out? I've been using Google Docs from the start so I'm going to keep using it and I'm game to share the link with my Room Leaders.
2. I **definitely** want to keep recording the intros! You learn interesting little anecdotes about people you ordinarily wouldn't through them!! As for the notes, the level of granularity doesn't matter to me. It seems that some notetakers may have forgotten to copy over the original directions into their template where in the main section it says: _"You can capture as much or as little as works for you. This could be a summary, bullet points, or detailed notes."_ Don't stress about it too much yall! 😃 
3. Yes, notetakers should do what they feel comfortable with. I believe that most will use the provided template and workflow since it's already set up for them, but as a volunteer role, it should be whatever works for you!

## Idea

I've been thinking (uh oh, I know... 😉 ) about creating a Slack bot to assign the coffee Room Leaders and Notetakers each week (one less thing yall have to remember to do). What if I took it a step further and had volunteers give a preference? Room Leaders could choose whether or not they prefer to receive a notes link/participate in light note taking and Notetakers could choose whether or not they want to use the Google Docs template. Then I can write a program to pair Room Leaders and Notetakers together according to their availability and preferences.

I wouldn't start working on this until July so there's plenty of time to think about it, discuss, and plan. Let me know what yall think!

**Comment by danieltott** [comment]
I personally feel like _less_ structure is really good here. It's cool to have a suggested template, but I actually really like that everyone has different note-taking styles - I feel like it adds to the flavor.

**Comment by clandau** [comment]
Giving my feedback.

**1. Do notetakers mostly want a google doc--which adds one more place to keep track of?**
Personally....not really. I like my super long Bear note that I just keep adding to. If we do decide that we need to share notes with the room leader, I'd be fine with using one. But I also don't feel like the room leader needs to help out with notes, their job keeping convos going can be challenging. I don't feel like the notes are that important. On the other hand I could see it being helpful if someone needs to step away or is getting chatted at by an 8yo off-screen 🙂. Anyway I think I'm all over the place here. Basically I like the way I am currently set up but happy to change if it suits the team. 

**2. Do we want to record all the topics and detailed notes or stick to intros, summaries, and links?**
I feel like we're already recording all the topics and doing detailed-enough notes. Sometimes the conversation is more opinion based, if that makes sense, (I'm thinking about one coffee which we answered questions like, "what time do you work best?") and keeping them anonymous while also writing those notes means the notes aren't as detailed. I guess what I want is "permission" for less detailed notes when the conversation goes in this direction. 

**3. Do we want to let the notetakers do what they feel comfortable with, whether that's using a shared template or YOLOing notes?**
That's up to you all. Maybe a poll of the entire community to see how people who read them regularly feel would be helpful. 

ALSO! I'm happy for note feedback, good and bad. If I'm deviating from the standard let me know and I'll change them. I'm not usually someone who regularly reads all the other rooms' notes but I want them to be good for those that do!

**Reply by meg-gutshall** [reply]
> or is getting chatted at by an 8yo off-screen

Now who could that be?? 🤔  😉

**Reply by tkshill** [reply]
I do the same Courtney, but with a dropbox file. It's super long now but always there and that's my go to.

**Comment by BekahHW** [comment]
1. "Another place to keep track of"- we're currently using github and slack. And then adding the template to google docs, which is a third place someone would need to keep track of. I think having the template in Github is a good point of reference so everyone know where the latest version of everything is and can keep tabs on any updates to the docs in general. I think it's fine for ppl to use google docs--I'm there all the time myself-- but I don't think it needs to be an official place for VC to house info right now.
2. I think that automating things can be really good in some circumstances, but there are also some reasons that it won't always work. For me, this is one of those things that still needs a human touch.

**Comment by tkshill** [comment]
So, do we all feel like some version of the following?
- We're probably keeping records of the things we want to in the notes, for the reasons we want to.
- The template is good, for reference and new folks and to establish some basic rules of order, but the fact that everyone has their own little spin and flair on notes is to be expected and encouraged
- The key point in the zoom discussion is we said it could be good for the note-takers to share their notes with the room leaders, and like, everyone is free to do so, but they don't have to. We're a team, and an intimate enough group that RLs and NTs can communicate their needs to each other.
